data/sea/52-psv.h

```diff
@@ -338,27 +343,20 @@ static ierror_loc_t psv_read_buffer (psv_state_t *s, const size_t facts_limit)
 
         if (new_entity < 0) {
             if (entity_cur_size != 0) {
```

Comment:
`entity_cur_size` is the old entity's size. I don't think I added this check, but it might have been to check against the first entity? will try removing it.